.github/workflows/github-release.yml

```diff
@@ -82,14 +82,17 @@ jobs:
       - name: ci/checkout-repo
         uses: actions/checkout@b4ffde65f46336ab88eb53be808477a3936bae11 # v4.1.1
 
-      - uses: ruby/setup-ruby@22fdc77bf4148f810455b226c90fb81b5cbc00a7 # v1.171.0
+      - uses: ruby/setup-ruby@ff740bc00a01b3a50fffc55a1071b1060eeae9dc # v1.180.0
 
       - name: release/setup-fastlane-dependencies
         run: bundle install
         working-directory: ./fastlane
 
       - name: ci/download-artifacts
-        uses: actions/download-artifact@9bc31d5ccc31df68ecc42ccf4149144866c47d8a # v3.0.2
+        uses: actions/download-artifact@65a9edc5881444af0b9093a5e628f2fe47ea3b2e # v4.1.7
+        with:
+          path: ${{ github.workspace }}
```

Comment:
Is this needed?

Reply:
I think yes if you want to download them in the same directory . 
https://github.com/actions/download-artifact?tab=readme-ov-file#download-all-artifacts
They have completely refactored v4